tutorials/IBC-connection.md

```diff
@@ -0,0 +1,283 @@
+cosmos-sdk: v0.47.5
+go: go1.21.4 darwin/arm64
+```
+
+### Setup relayer config
+
+Firstly, generate an empty config file with this command:
+
+```sh
+rly config init
+```
+
+Afterward, edit the file in the directory `$HOME/.relayer/config/config.yaml` with any text editor you prefer. Then, paste this configuration into it.
+
+```config
+global:
+    api-listen-addr: :5183
+    timeout: 10s
+    memo: ""
+    light-cache-size: 20
+chains:
+    osmo-local:
+        type: cosmos
+        value:
+            key-directory: /root/.relayer/keys/osmosis-testnet-1
+            key: default
+            chain-id: osmosis-testnet-1
+            rpc-addr: http://localhost:46657 
+            account-prefix: osmo
+            keyring-backend: test
+            gas-adjustment: 1.5
+            gas-prices: 0.1uosmo
+            min-gas-amount: 100000
+            max-gas-amount: 1000000
+            debug: true
+            timeout: 20s
+            block-timeout: ""
+            output-format: json
+            sign-mode: direct
+            extra-codecs: []
+            coin-type: 118
+            signing-algorithm: ""
+            broadcast-mode: batch
+            min-loop-duration: 0s
+            extension-options: []
+            feegrants: null
+    gm-local:
+        type: cosmos
+        value:
+            key-directory: /root/.relayer/keys/test
+            key: default
+            chain-id: gm
+            rpc-addr: http://localhost:36657
+            account-prefix: gm
+            keyring-backend: test
+            gas-adjustment: 1.5
+            gas-prices: 0.1stake
+            min-gas-amount: 100000
+            max-gas-amount: 1000000
+            debug: true
+            timeout: 20s
+            block-timeout: ""
+            output-format: json
+            sign-mode: direct
+            extra-codecs: []
+            coin-type: 118
+            signing-algorithm: ""
+            broadcast-mode: batch
+            min-loop-duration: 0s
+            extension-options: []
+            feegrants: null
+paths: {}
+```
+
+### Create relayer account
+
+Add keys for each chain with this command
+
+The mnemonic-words is the mnemonic that you have when init local node, ensure that each wallet have token to start relaying.
+
+```sh
+rly keys restore osmo-local default "milk verify alley price trust come maple will suit hood clay exotic"
+rly keys restore gm-local default "milk verify alley price trust come maple will suit hood clay exotic"
+```
+
+### Create IBC channel
+
+Create a new blank path to be used in generating a new path (connection and client) between two chains
+
+```sh
+rly paths new osmosis-testnet-1 gm osmo-gm
+```
+
+and then you can create channel with this command
+
+```sh
+rly transact link osmo-gm
+```
+
+This is a triplewammy, it creates a client, connection, and channel all in one command.
+
+Alternatively, you may create them one by one using these commands:
+
+```sh
+rly transact clients osmo-local gm-local osmo-gm
```

Comment:
yes, i am blocked on https://github.com/rollkit/docs/pull/324#discussion_r1494531949

Reply:
@gupadhyaya @jcstein we need to replace rollkit version by a commit hash 
`go mod edit -replace=github.com/rollkit/rollkit=github.com/rollkit/rollkit@03f631c2888055979d83a758244701c7d44a0083` because lastest release (v0.11.19) doesn't support IBC

Reply:
should we wait to new rollkit version or just add this replace cmd into docs ?

Reply:
i am doing local repo replace which has IBC support, still have the error. any idea? @vuong177

Reply:
I think it's from ignite version, could you upgrade ignite version to `v28.2.0`

Reply:
we need to upgrade ignite version in docs too

Reply:
I tried with the 2 ignite version. I got same error as yours with ignite v28.1.0 but with new version (v28.2.0), everything working well. Could you please check it agin ?

Reply:
working ignite version :
```
ignite version
Ignite CLI version:		v28.2.0
Ignite CLI build date:		2024-02-06T11:21:56Z
Ignite CLI source hash:		af25183a2d312aa4d679e65e3d627025b17a7120
Ignite CLI config version:	v1
Cosmos SDK version:		v0.50.3
Your OS:			darwin
Your arch:			arm64
Your Node.js version:		v20.11.0
Your go version:		go version go1.21.4 darwin/arm64
Your uname -a:			Darwin 192.168.0.224 23.2.0 Darwin Kernel Version 23.2.0: Wed Nov 15 21:54:05 PST 2023; root:xnu-10002.61.3~2/RELEASE_ARM64_T6031 arm64
Your cwd:			/Users/vuongnguyen
Is on Gitpod:			false
```

Reply:
resolving this as the ignite version is in top of tutorial